lib/cli.js

```diff
@@ -27,7 +27,8 @@ updateNotifier({ pkg }).notify();
 
 const cli = () => {
   const cwd = process.cwd();
-  const latestTaggedCommit = taggedCommit({ path: cwd });
+  let latestTaggedCommit = taggedCommit({ path: cwd }); // returns an array
+  latestTaggedCommit = latestTaggedCommit.length === 0 ? '' : latestTaggedCommit[0];
```

Comment:
I think the naming is quite confusing. First it is an array, and then it is just one commit? I would prefer to not reassign the variables and write it like `const latestTaggedCommits` and `const latestTaggedCommit`. But it is also good as is.